src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml/Given_FrameworkElement_And_Leak.cs

```diff
@@ -50,7 +50,7 @@ public class Given_FrameworkElement_And_Leak
 		[DataRow(typeof(ListView), 15)]
 		[DataRow(typeof(ProgressRing), 15)]
 		[DataRow(typeof(Pivot), 15)]
-		// [DataRow(typeof(ScrollBar), 15)] https://github.com/unoplatform/uno/issues/7331
+		[DataRow(typeof(ScrollBar), 15)] // https://github.com/unoplatform/uno/issues/7331
```

Comment:
If it works, let's remove this:
```suggestion
		[DataRow(typeof(ScrollBar), 15)]
```